tools/objc/libtool.sh

```diff
@@ -32,15 +32,22 @@ if [ -z ${MY_LOCATION+x} ]; then
   fi
 fi
 
-WRAPPER="${MY_LOCATION}/xcrunwrapper.sh"
+function libtool() {
```

Comment:
probably worth naming this something slightly different to avoid confusion that we're calling `libtool` directly, maybe `libtool_func` (or ideally something better)

Reply:
Would `invoke_libtool` be okay?

Reply:
sounds good